new/resource-current-value.md

```diff
@@ -0,0 +1,98 @@
+---
+RFC: unassigned
+Author: John Keiser <jkeiser@chef.io
+Status: Draft
+Type: Standards Track
+---
+
+# Allow Current Value To Be Read From Resource
```

Comment:
The intent is to read the actual value from the system, not from the resource collection, actually.